### ISSUE TITLE
Legend unselectedOpacity only works via config

### ISSUE THREAD

**Issue by jwoLondon** [vega/vega-lite#5921]
The [legend documentation](https://vega.github.io/vega-lite/docs/legend.html#properties) implies that `unselectedOpacity` can be set directly as a property of legend encoding, but it appears not be part of the schema and has no effect.

It does work when specified as part of a config though:

```json
"config": {"legend": {"unselectedOpacity": 0.01}}
```

Is the intention to allow it also to be set directly as a property? If not, the documentation should reflect this.

**Comment by domoritz** [comment]
I agree that it should work as part of the legend as well. Do you agree @arvind ?
